FND/DatesAndTimes/FinancialDates.rdf

```diff
@@ -252,7 +252,7 @@ The nth day is an ordinal number, not a cardinal number. &apos;1&apos; means the
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>date time</rdfs:label>
-		<skos:definition>a combination of a date and a time, without a time zone</skos:definition>
+		<skos:definition>combination of a date and a time, without a time zone</skos:definition>
```

Comment:
If the TZ does not come from this, where does it come from? Without such a statement there is unacceptable ambiguity.